### DISCUSSION TITLE
Block parameters by device_type that should not create entities in HA.

### DISCUSSION THREAD

**Discussion by SukramJ** [SukramJ/aiohomematic#858] [General]
Feature request:

This integration is based on static meta data like`paramset description` and `device descriptions` that are return by a CCU.

Problem:
The meta data are not well engineered by eq-3.
- `LOWBAT`/`LOW_BAT`/`OPERATING_VOLTAGE`: some devices have battery related parameters although they are operated by mains
<del> - `RSSI_PEER`: most battery operated devices don't return RSSI_PEER values, or this happens on a very irregular or random manner.  </del>

Solution:
- Add option to block parameters
- Collect parameters for device_types that should not create entities. 

Try to avoid duplicate entries.

**Comment by SukramJ** [comment]
LOWBAT:
HM-LC-Sw4-DR, HM-SwI-3-FM, HM-LC-Sw1PBU-FM, HM-LC-Sw1-FM

LOW_BAT:
HmIP-BWTH

OPERATING_VOLTAGE:
HmIP-BWTH, HmIP-BSM, HmIP-BDT, HmIP-PS*, HmIP-PDT, HmIP-FSM, HmIP-FDT, HmIP-SFD

**Comment by Baxxy13** [comment]
LOWBAT:
HM-LC-Sw1-Pl-DN-R1, HM-LC-Sw1-PCB,  

OPERATING_VOLTAGE:
HmIP-DRSI1, HmIP-DRSI4, HmIP-PCBS, HmIP-PCBS2, HmIP-MOD-OC8

Ich habe ein kleines Homematic-Script erstellt falls jemand sein System prüfen möchte.
Einfach in der ersten Zeile den gesuchten Parameternamen eingeben. 
Die Auswertung muss dann aber menschliche Intelligenz erledigen und die (möglicherweise) gelisteten CUxD-Geräte können ignoriert werden.

Script: ("Script testen" auf CCUx / RM)
```
string auswahl = "LOW_BAT";
! +++++++++++++++++++
string index;
string liste = dom.GetObject(ID_DATAPOINTS).EnumUsedNames();
foreach (index, liste) {
var sserial = index.StrValueByIndex(".",2);;
if (sserial == auswahl) {
integer kanalnummer = (dom.GetObject(dom.GetObject(index).Channel())).ChnNumber();
if (!kanalnummer) { string dev = (dom.GetObject(dom.GetObject(index).Channel())).HssParentType();
Write(dev # " - " # index #"\n");}}}
```

Beispielausgabe:
```
HmIP-HEATING - VirtualDevices.INT0000001:0.LOW_BAT
HmIP-MOD-RC8 - HmIP-RF.00145709AED7F9:0.LOW_BAT
HmIP-ASIR - HmIP-RF.000AD8A99E7315:0.LOW_BAT
HmIP-SCI - HmIP-RF.001E1D898A3967:0.LOW_BAT
```

**Comment by SukramJ** [comment]
HmIP-HEATING und HM-CC-VG-1 sollten eigentlich nur eine climate Entität enthalten.

>LOWBAT:
HM-LC-Sw1-Pl-DN-R1, HM-LC-Sw1-PCB,

Lässt sich das auf `HM-LC-Sw*` verallgemeinern?

**Comment by Baxxy13** [comment]
Du warst zu schnell mit lesen. Habe den Passus mit den Heizgruppen wieder herausgenommen, da ich das nicht nachgeprüft hatte.

 > verallgemeinern?

Kann ich nicht sagen, habe nur die beiden.

**Comment by SukramJ** [comment]
Dann verallgemeinere ich da mal lieber nicht.

**Comment by meks007** [comment]
Ich würde zu dem LOW_BAT gerne den HmIP-MP3P hinzufügen. Der MP3P hat 2 Modi - Netzbetrieb und Batteriebetrieb, das kann man auch in den Einstellungen umstellen. LOW_BAT wird aber trotzdem im HA angelegt - Status immer Normal. Unkritisch, wollte es aber erwähnen.

**Comment by SukramJ** [comment]
In diesem Fall würde ich empfehlen die Entität einfach in HA zu deaktivieren.

**Comment by ChristophCaina** [comment]
HmIP-Broll => provides Operating Voltage
![grafik](https://user-images.githubusercontent.com/26391061/211541939-bf77063b-379f-4736-8625-a9e9ebb2452d.png)
